compiler/src/iree/compiler/Dialect/VMVX/IR/VMVXOps.td

```diff
@@ -82,15 +82,17 @@ def VMVX_QueryTileSizesOp : VMVX_PureOp<"query_tile_sizes"> {
   }];
 
   let arguments = (ins
-    I64:$encoding,
-    Variadic<VMVX_Index>:$sizes
+    Variadic<VMVX_Index>:$sizes,
+    I32Attr:$flags
   );
   let results = (outs
     Variadic<VMVX_Index>:$tileSizes
   );
 
   let assemblyFormat = [{
-    `encoding` `(` $encoding `)` `sizes` `(` $sizes `)` `->` type(results) attr-dict
+    `sizes` `(` $sizes `)`
```

Comment:
```suggestion
    `sizes` `` `(` $sizes `)`
```
an empty backtick pair will elide the space (so we would have `sizes(%foo)` instead of `sizes (%foo)`

Reply:
Thanks, I didn't know. Done.